Dockerfile

```diff
@@ -1,32 +1,35 @@
 # syntax=docker/dockerfile:experimental
 
-# Build stage: Install ruby dependencies
-# ===
-FROM ruby:2.5 AS build-site
-WORKDIR /srv
-ADD . .
-RUN bundle install
-RUN bundle exec jekyll build
-
 # Build stage: Install yarn dependencies
 # ===
-FROM node:12-slim AS yarn-dependencies
+FROM node:13-slim AS yarn-dependencies
 WORKDIR /srv
 ADD package.json .
 RUN --mount=type=cache,target=/usr/local/share/.cache/yarn yarn install
 
 # Build stage: Run "yarn run build-js"
 # ===
 FROM yarn-dependencies AS build-js
-ADD . .
+WORKDIR /srv
+COPY src .
 RUN yarn run build-js
 
 # Build stage: Run "yarn run build-css"
 # ===
 FROM yarn-dependencies AS build-css
-ADD . .
+WORKDIR /srv
+COPY src/_sass src/_sass
```

Comment:
This probably wasn't changed by you - but the underscore here suggests the sass isn't included in the eventual site. It should be, because we probably want to use sourcemaps to tell the browser which lines of CSS were generated from which lines of Sass. it can't do this unless the Sass files are also published.

Basically, this directory should be called `sass` rather than `_sass`, and should be included in the built site. But you don't have to change this in this PR if you don't have time.